vendors/espressif/boards/esp32/CMakeLists.txt

```diff
@@ -410,14 +410,18 @@ target_compile_definitions(
 endif()
 
 # OTA
-# Need to get this validated
 afr_mcu_port(ota)
 target_sources(
     AFR::ota::mcu_port
     INTERFACE
-        "${afr_ports_dir}/ota/aws_ota_pal.c"
-        "${afr_ports_dir}/ota/aws_esp_ota_ops.c"
-        "${afr_ports_dir}/ota/aws_esp_ota_ops.h"
+        "${afr_ports_dir}/ota_pal_for_aws/ota_pal.c"
+        "${afr_ports_dir}/ota_pal_for_aws/aws_esp_ota_ops.c"
+        "${afr_ports_dir}/ota_pal_for_aws/aws_esp_ota_ops.h"
+)
+target_include_directories(
+    AFR::ota::mcu_port
+    INTERFACE
+        "${afr_ports_dir}/ota_pal_for_aws"
 )
 target_include_directories(
     AFR::ota::mcu_port
```

Comment:
Is there are a reason to have two separate taret_include_directories calls for the same module? If not, can you merge these into one?

Reply:
Good catch, merged into one.